src/main/java/ru/skypro/homework/entity/Comment.java

```diff
@@ -0,0 +1,99 @@
+package ru.skypro.homework.entity;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+public class Comment {
+
+    @javax.persistence.Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE)
+    private int pk;
+
+    @ManyToOne
+    @JoinColumn(name = "author")
+    private int author;
```

Comment:
здесь должен быть тип данных User:

```suggestion
    private User author;
```